hello_world/urls.py

```diff
@@ -3,9 +3,11 @@
 from hello_world import api
 
 urlpatterns = [
+    path('guests_endpoint/', api.guest_list_endpoint, name='guest-list-endpoint'),
```

Comment:
Json data paths should start with the word 'api'

Reply:
Corrected